app-text/multimarkdown/multimarkdown-6.7.0.recipe

```diff
@@ -70,6 +70,10 @@ BUILD()
 INSTALL()
 {
 	make -C build install
+
+	for f in markdown mmd mmd2all mmd2epub mmd2fodt mmd2odt mmd2opml mmd2pdf mmd2tex ;
```

Comment:
Just a small syntactic nitpick (this looks a bit strange): either remove the `;` at the end or move the `do` at the end of this line (I would prefer the latter).

Reply:
Done, thanks!